web-app/src/app/tabs/controls/controls.component.css

```diff
@@ -0,0 +1,82 @@
+.alive-group {
+	margin-left: auto;
+}
+.aliveGroup > .aliveText {
+	margin-left: 0.2em;
+}
+.circleAlive {
+	float: left;
+	width: 1em;
+	height: 1em;
+	margin-top: 2px;
+	margin-right: .4em;
+}
+.width {
+	width: 100%;
+}
+.spacing {
+	width: 100%;
+	text-align: left;
+	font-size: 1.2rem;
+	padding-bottom: 0.2em;
+	font-family: Ariel, sans-serif;
+}
+.dataHeader {
+	color: rgba(255, 255, 255, 0.75);
+}
+.titleUnderline {
+	border-bottom: 1px solid white;
+	margin-bottom: 5%;
+}
+.dataUnderline {
+	border-bottom: 1px solid white;
+}
+.on {
+	color: #c2185b;
+}
+.off {
+	color: grey;
+}
+.lightIcon {
+	width: 35px;
+	padding-top: 10%;
+}
+.signalSpacing {
+	width: 5px;
+}
+.container-row {
+	display: flex;
+	flex-direction: row;
+	justify-content: center;
+}
+.container-row-not-centered {
+	display: flex;
+	flex-direction: row;
+}
+.container-column {
```

Comment:
Use the `centered` style I mentioned above and remove that line. 

For `.container-column-data`, make a `flex-start` style and invoke with `container-column flex-start`